client/src/components/questions/ImageModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { StyledBackground } from './AnswerModal.jsx';
+import styled from 'styled-components';
+
+const StyledImage = styled.img`
+  width: auto;
+  height: auto;
+`;
+
+const StyledExitButton = styled.button`
+  position: absolute;
+  top: 0; left: 0;
+  height: 5vh;
+  width: 5vw;
+  cursor: pointer;
+`;
+
+const StyledImageModal = styled.div`
+  position: fixed;
+  top: 12.5vh; left: 25vw;
+  z-index: 101;
+  max-width: 50vw;
```

Comment:
Doesn't a high z-index send an element to the back? Don't you want the modal to be in the front?

Reply:
higher z-index means higher priority to be in the forefront of the page